.github/workflows/bot-executable.yml

```diff
@@ -0,0 +1,141 @@
+name: Build bot executable file
+
+on:
+  push:
+    branches:
+      - 'develop'
+      - 'master'
+      - 'release/**'
+      - 'hotfix/**'
+      - 'feature/RPA-1097' # todo remove before merge to develop
+
+env:
+  NODE_VERSION: 18.x
+  PNPM_VERSION: 8.7.x
+  RUSH: node ../common/scripts/install-run-rush.js
+
+jobs:
+  changes-detection:
+    runs-on: windows-latest
+
+    outputs:
+      bot: ${{ steps.filter.outputs.bot }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Assign paths to scenarios
+        uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            bot:
+              - 'runbotics/runbotics-desktop/package.json'
+  build-bot:
+    needs: changes-detection
+    if: ${{ needs.changes-detection.outputs.bot == 'true' }}
+    runs-on: windows-latest
+
+    defaults:
+      run:
+        working-directory: ./runbotics/runbotics-desktop
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Clone submodules
+        run: git submodule update --init --recursive
```

Comment:
why do you need it here?

Reply:
without it the signing/electron-builder wasn't working: ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In ..\common\deploy\runbotics-desktop\runbotics-desktop: "runbotics-common@workspace:^1.0.0" is in the dependencies but no package named "runbotics-common" is present in the workspace